### ISSUE TITLE
sigmoid_cross_entropy Docstring bug

### ISSUE THREAD

**Issue by Sarasra** [tensorflow/tensorflow#17021]
### Describe the problem
Tf.losses.sigmoid_cross_entory parameter description of label indicates it has to be "integer" and in range (0,1). There are no integers between (0,1) so that seems really difficult to abide by. 
It seems that in the code it does not need to be integer and the range should be [0, 1]. 
https://www.tensorflow.org/api_docs/python/tf/losses/sigmoid_cross_entropy

P.S. Why does softmax_cross_entropy assume one-hot encoding? For instance in distillation your labels are softmax outputs (floats in [0, 1]).

**Comment by aidangomez** [comment]
+1 This parameter description was hilariously confusing.

**Comment by bignamehyp** [comment]
@drpngx  can you please take a look?

**Comment by drpngx** [comment]
From the code, it looks like it just wants binary labels. I read the doc to mean either `0` or `1`. Feel free to send a PR to clarify the doc.
